optimum/intel/openvino/modeling.py

```diff
@@ -371,23 +373,27 @@ class OVModelForFeatureExtraction(OVModel):
 
     def __init__(self, model=None, config=None, **kwargs):
         super().__init__(model, config, **kwargs)
+        
+        from sentence_transformers import SentenceTransformer
```

Comment:
I suppose original model class should not depends from sentence transformers. Can we create separated model class OVModelForSentenceTransformers that will have this functionality. We should keep default model class from models exported with library_name="transformers" and have separated one for exported with "sentence_transformers". Both model variants are valid and should work, now you override to have only sentence transformers.